xml/decoders/lenz/functionmap.xml

```diff
@@ -66,195 +72,222 @@
     <label>F4 controls output A</label>
     <label xml:lang="it">F4 controlla uscita A</label>
     <label xml:lang="de">F4 steuert Ausgang A</label>
     <label xml:lang="cs">F4 funkční výstup A</label>
+    <label xml:lang="ca">F4 controla la sortida A</label>
   </variable>
   <variable CV="39" mask="XXXXXXXV" item="F5 controls output 1" minOut="1" minFn="5">
     <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
     <label>F5 controls output A</label>
     <label xml:lang="it">F5 controlla uscita A</label>
     <label xml:lang="de">F5 steuert Ausgang A</label>
     <label xml:lang="cs">F5 funkční výstup A</label>
+    <label xml:lang="ca">F5 controla la sortida A</label>
   </variable>
   <variable CV="40" mask="XXXXXXXV" item="F6 controls output 1" minOut="1" minFn="6">
     <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
     <label>F6 controls output A</label>
     <label xml:lang="it">F6 controlla uscita A</label>
     <label xml:lang="de">F6 steuert Ausgang A</label>
     <label xml:lang="cs">F6 funkční výstup A</label>
+    <label xml:lang="ca">F6 controla la sortida A</label>
   </variable>
   <variable CV="41" mask="XXXXXXXV" item="F7 controls output 1" minOut="1" minFn="7">
     <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
     <label>F7 controls output A</label>
     <label xml:lang="it">F7 controlla uscita A</label>
     <label xml:lang="de">F7 steuert Ausgang A</label>
     <label xml:lang="cs">F7 funkční výstup A</label>
+    <label xml:lang="ca">F7 controla la sortida A</label>
   </variable>
   <variable CV="42" mask="XXXXXXXV" item="F8 controls output 1" minOut="1" minFn="8">
     <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
     <label>F8 controls output A</label>
     <label xml:lang="it">F8 controlla uscita A</label>
     <label xml:lang="de">F8 steuert Ausgang A</label>
     <label xml:lang="cs">F8 funkční výstup A</label>
+    <label xml:lang="ca">F8 controla la sortida A</label>
   </variable>
   <variable CV="33" mask="XXXVXXXX" item="FL(f) controls output 2" minOut="2" minFn="0">
     <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
     <label>F0(f) controls output B</label>
     <label xml:lang="it">F0(avanti) controlla uscita B</label>
     <label xml:lang="de">F0(f) steuert Ausgang B</label>
     <label xml:lang="cs">F0 vpřed funkční výstup B</label>
+    <label xml:lang="ca">F0 (davant) controla la sortida A</label>
   </variable>
   <variable CV="34" mask="XXXVXXXX" item="FL(r) controls output 2" default="1" minOut="2" minFn="0">
     <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
     <label>F0(r) controls output B</label>
     <label xml:lang="it">F0(indietro) controlla uscita B</label>
     <label xml:lang="de">F0(r) steuert Ausgang B</label>
-    <label xml:lang="cs">F0 vzad funkční výstup B</label>
```

Comment:
The removal of this line looks to be a mistake - the Czech translation should remain